bookserver/routers/auth.py

```diff
@@ -109,6 +109,15 @@ async def login(
     return response
 
 
+# To log out, simply delete the cookie containing auth information.
+@router.get("/logout")
```

Comment:
Nit: maybe add response_class: RedirectResponse

Reply:
Thanks! I'll add this.